teamcity: failed test: TestDistAggregationTable

The following tests appear to have failed on master (testrace): TestDistAggregationTable/AVG/dec1/50, TestDistAggregationTable/MIN/dec1/50, TestDistAggregationTable/MIN/dec1/100, TestDistAggregationTable/STDDEV/float1/5, TestDistAggregationTable/COUNT/dec1/5, TestDistAggregationTable/MAX/bool2/50, TestDistAggregationTable/SUM/int1/5, TestDistAggregationTable/STDDEV/float1/50, TestDistAggregationTable, TestDistAggregationTable/BOOL_OR/bool1/10, TestDistAggregationTable/STDDEV/int2/10, TestDistAggregationTable/MAX/bool1/10, TestDistAggregationTable/MIN/dec2/50, TestDistAggregationTable/MIN/float1/50, TestDistAggregationTable/STDDEV/dec1/5, TestDistAggregationTable/COUNT/dec2/10, TestDistAggregationTable/STDDEV/float1/10, TestDistAggregationTable/MAX/bool1/5, TestDistAggregationTable/MAX/b/100, TestDistAggregationTable/SUM/dec2/50, TestDistAggregationTable/BOOL_OR/bool2/100, TestDistAggregationTable/MIN/float1/10, TestDistAggregationTable/SUM/int1/50, TestDistAggregationTable/AVG/int2/10, TestDistAggregationTable/COUNT/int1/50, TestDistAggregationTable/MIN/int2/5, TestDistAggregationTable/MAX/int2/100, TestDistAggregationTable/AVG/int1/50, TestDistAggregationTable/STDDEV/dec2/50, TestDistAggregationTable/SUM/float1/50, TestDistAggregationTable/MAX/dec1/50, TestDistAggregationTable/COUNT/b/50, TestDistAggregationTable/AVG/float2/100, TestDistAggregationTable/SUM/float2/50, TestDistAggregationTable/COUNT/b/100, TestDistAggregationTable/SUM/float1/100, TestDistAggregationTable/MIN/bool2/10, TestDistAggregationTable/MAX/int2/5, TestDistAggregationTable/AVG/float2/50, TestDistAggregationTable/SUM/float2/100, TestDistAggregationTable/MAX/dec2/100, TestDistAggregationTable/STDDEV/int2/50, TestDistAggregationTable/MAX/dec1/10, TestDistAggregationTable/COUNT/float1/100, TestDistAggregationTable/MIN/bool2/50, TestDistAggregationTable/MAX/bool2/10, TestDistAggregationTable/COUNT/bool1/5, TestDistAggregationTable/COUNT/b/5, TestDistAggregationTable/BOOL_OR/bool1/100, TestDistAggregationTable/COUNT/int1/5, TestDistAggregationTable/STDDEV/float2/50, TestDistAggregationTable/MAX/b/10, TestDistAggregationTable/STDDEV/int1/5, TestDistAggregationTable/COUNT/bool2/10, TestDistAggregationTable/STDDEV/int1/100, TestDistAggregationTable/STDDEV/dec2/100, TestDistAggregationTable/COUNT/bool2/100, TestDistAggregationTable/COUNT/dec1/50, TestDistAggregationTable/MAX/bool1/50, TestDistAggregationTable/MAX/float2/50, TestDistAggregationTable/AVG/dec2/100, TestDistAggregationTable/COUNT/int1/100, TestDistAggregationTable/BOOL_AND/bool2/10, TestDistAggregationTable/SUM/int2/10, TestDistAggregationTable/AVG/float1/100, TestDistAggregationTable/BOOL_OR/bool1/50, TestDistAggregationTable/MAX/float1/10, TestDistAggregationTable/MIN/int2/100, TestDistAggregationTable/BOOL_AND/bool1/10, TestDistAggregationTable/MAX/int1/100, TestDistAggregationTable/SUM/int2/100, TestDistAggregationTable/SUM/dec2/5, TestDistAggregationTable/COUNT/dec2/5, TestDistAggregationTable/STDDEV/dec2/10, TestDistAggregationTable/MIN/bool1/50, TestDistAggregationTable/MIN/float1/100, TestDistAggregationTable/STDDEV/float2/5, TestDistAggregationTable/AVG/int1/10, TestDistAggregationTable/MIN/float2/5, TestDistAggregationTable/SUM/int2/5, TestDistAggregationTable/MIN/dec2/5, TestDistAggregationTable/COUNT/float2/5, TestDistAggregationTable/STDDEV/float2/100, TestDistAggregationTable/COUNT/float2/10, TestDistAggregationTable/SUM/int2/50, TestDistAggregationTable/STDDEV/float2/10, TestDistAggregationTable/STDDEV/dec1/100, TestDistAggregationTable/AVG/float1/5, TestDistAggregationTable/SUM/float2/5, TestDistAggregationTable/SUM/float1/10, TestDistAggregationTable/COUNT/int2/50, TestDistAggregationTable/MAX/bool2/100, TestDistAggregationTable/MAX/float1/50, TestDistAggregationTable/MAX/int2/50, TestDistAggregationTable/MAX/float2/100, TestDistAggregationTable/COUNT/bool2/5, TestDistAggregationTable/MAX/int1/10, TestDistAggregationTable/COUNT/int2/10, TestDistAggregationTable/AVG/float2/5, TestDistAggregationTable/SUM/float2/10

You may want to check [for open issues](https://github.com/cockroachdb/cockroach/issues?q=is%3Aissue+is%3Aopen+TestDistAggregationTable).

[#1013127](https://teamcity.cockroachdb.com/viewLog.html?buildId=1013127):


```
TestDistAggregationTable/AVG/float2/100
--- FAIL: testrace/TestDistAggregationTable/AVG/float2/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/BOOL_AND/bool1/10
--- FAIL: testrace/TestDistAggregationTable/BOOL_AND/bool1/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/STDDEV/float2/50
--- FAIL: testrace/TestDistAggregationTable/STDDEV/float2/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/COUNT/bool2/10
--- FAIL: testrace/TestDistAggregationTable/COUNT/bool2/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/MAX/bool1/10
--- FAIL: testrace/TestDistAggregationTable/MAX/bool1/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/MAX/bool2/100
--- FAIL: testrace/TestDistAggregationTable/MAX/bool2/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/AVG/float2/5
--- FAIL: testrace/TestDistAggregationTable/AVG/float2/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/MIN/dec1/100
--- FAIL: testrace/TestDistAggregationTable/MIN/dec1/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/COUNT/b/50
--- FAIL: testrace/TestDistAggregationTable/COUNT/b/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/STDDEV/int2/50
--- FAIL: testrace/TestDistAggregationTable/STDDEV/int2/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/COUNT/b/100
--- FAIL: testrace/TestDistAggregationTable/COUNT/b/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/COUNT/dec2/10
--- FAIL: testrace/TestDistAggregationTable/COUNT/dec2/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/AVG/int1/10
--- FAIL: testrace/TestDistAggregationTable/AVG/int1/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/MAX/dec2/100
--- FAIL: testrace/TestDistAggregationTable/MAX/dec2/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/BOOL_OR/bool2/100
--- FAIL: testrace/TestDistAggregationTable/BOOL_OR/bool2/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/COUNT/int2/10
--- FAIL: testrace/TestDistAggregationTable/COUNT/int2/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/MIN/bool2/50
--- FAIL: testrace/TestDistAggregationTable/MIN/bool2/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/MAX/int2/100
--- FAIL: testrace/TestDistAggregationTable/MAX/int2/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/COUNT/dec2/5
--- FAIL: testrace/TestDistAggregationTable/COUNT/dec2/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/MIN/float2/5
--- FAIL: testrace/TestDistAggregationTable/MIN/float2/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/MIN/dec2/5
--- FAIL: testrace/TestDistAggregationTable/MIN/dec2/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/MIN/bool1/50
--- FAIL: testrace/TestDistAggregationTable/MIN/bool1/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/SUM/int1/50
--- FAIL: testrace/TestDistAggregationTable/SUM/int1/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/COUNT/float2/5
--- FAIL: testrace/TestDistAggregationTable/COUNT/float2/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/BOOL_OR/bool1/100
--- FAIL: testrace/TestDistAggregationTable/BOOL_OR/bool1/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/COUNT/dec1/50
--- FAIL: testrace/TestDistAggregationTable/COUNT/dec1/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/SUM/int2/100
--- FAIL: testrace/TestDistAggregationTable/SUM/int2/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/MAX/bool1/5
--- FAIL: testrace/TestDistAggregationTable/MAX/bool1/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/COUNT/bool2/100
--- FAIL: testrace/TestDistAggregationTable/COUNT/bool2/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/STDDEV/int2/10
--- FAIL: testrace/TestDistAggregationTable/STDDEV/int2/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/BOOL_OR/bool1/10
--- FAIL: testrace/TestDistAggregationTable/BOOL_OR/bool1/10 (0.000s)
Test ended in panic.

------- Stdout: -------
I181115 02:09:02.933994 1404 util/protoutil/randnullability.go:94  inserting null for (roachpb.Lease).Expiration: false
I181115 02:09:02.934150 1404 util/protoutil/randnullability.go:94  inserting null for (roachpb.Lease).DeprecatedStartStasis: true



TestDistAggregationTable/STDDEV/dec2/50
--- FAIL: testrace/TestDistAggregationTable/STDDEV/dec2/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/MAX/float1/50
--- FAIL: testrace/TestDistAggregationTable/MAX/float1/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/MAX/bool2/10
--- FAIL: testrace/TestDistAggregationTable/MAX/bool2/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/MIN/bool2/10
--- FAIL: testrace/TestDistAggregationTable/MIN/bool2/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/SUM/int2/10
--- FAIL: testrace/TestDistAggregationTable/SUM/int2/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/MAX/b/10
--- FAIL: testrace/TestDistAggregationTable/MAX/b/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/MIN/float1/10
--- FAIL: testrace/TestDistAggregationTable/MIN/float1/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/MAX/int1/100
--- FAIL: testrace/TestDistAggregationTable/MAX/int1/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/MAX/dec1/10
--- FAIL: testrace/TestDistAggregationTable/MAX/dec1/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/COUNT/dec1/5
--- FAIL: testrace/TestDistAggregationTable/COUNT/dec1/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/AVG/int2/10
--- FAIL: testrace/TestDistAggregationTable/AVG/int2/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/SUM/dec2/5
--- FAIL: testrace/TestDistAggregationTable/SUM/dec2/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/STDDEV/float2/10
--- FAIL: testrace/TestDistAggregationTable/STDDEV/float2/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/SUM/int2/5
--- FAIL: testrace/TestDistAggregationTable/SUM/int2/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/MAX/bool1/50
--- FAIL: testrace/TestDistAggregationTable/MAX/bool1/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/AVG/int1/50
--- FAIL: testrace/TestDistAggregationTable/AVG/int1/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/STDDEV/dec2/100
--- FAIL: testrace/TestDistAggregationTable/STDDEV/dec2/100 (0.000s)
Test ended in panic.

------- Stdout: -------
I181115 02:09:10.175250 4375 util/mon/bytes_usage.go:608  [n1] flow: bytes usage increases to 1.0 MiB (+10240)
I181115 02:09:10.272827 4376 util/mon/bytes_usage.go:608  [n1] flow: bytes usage increases to 1.0 MiB (+10240)
I181115 02:09:10.278828 4355 util/mon/bytes_usage.go:608  [n1] flow: bytes usage increases to 1.0 MiB (+10240)
I181115 02:09:10.279307 4380 util/mon/bytes_usage.go:608  [n1] flow: bytes usage increases to 1.0 MiB (+10240)



TestDistAggregationTable/MAX/b/100
--- FAIL: testrace/TestDistAggregationTable/MAX/b/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/SUM/float1/50
--- FAIL: testrace/TestDistAggregationTable/SUM/float1/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/COUNT/int1/5
--- FAIL: testrace/TestDistAggregationTable/COUNT/int1/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/COUNT/int2/50
--- FAIL: testrace/TestDistAggregationTable/COUNT/int2/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/AVG/dec1/50
--- FAIL: testrace/TestDistAggregationTable/AVG/dec1/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/STDDEV/dec2/10
--- FAIL: testrace/TestDistAggregationTable/STDDEV/dec2/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/MAX/int2/5
--- FAIL: testrace/TestDistAggregationTable/MAX/int2/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/SUM/float1/10
--- FAIL: testrace/TestDistAggregationTable/SUM/float1/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/SUM/float2/5
--- FAIL: testrace/TestDistAggregationTable/SUM/float2/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/STDDEV/int1/100
--- FAIL: testrace/TestDistAggregationTable/STDDEV/int1/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/MIN/int2/5
--- FAIL: testrace/TestDistAggregationTable/MIN/int2/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/MAX/dec1/50
--- FAIL: testrace/TestDistAggregationTable/MAX/dec1/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/STDDEV/float2/5
--- FAIL: testrace/TestDistAggregationTable/STDDEV/float2/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/SUM/float2/10
--- FAIL: testrace/TestDistAggregationTable/SUM/float2/10 (0.000s)
Test ended in panic.




TestDistAggregationTable
...:56.751191 12 server/testserver.go:426  had 12 ranges at startup, expected 20
I181115 02:08:56.751674 12 server/testserver.go:426  had 12 ranges at startup, expected 20
I181115 02:08:56.752216 12 server/testserver.go:426  had 12 ranges at startup, expected 20
I181115 02:08:56.766091 429 sql/event_log.go:126  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:46521} Attrs: Locality: ServerVersion:2.1-3 BuildTag:v2.2.0-alpha.00000000-2339-g90c65bc StartedAt:1542247734333485131 LocalityAddress:[]} ClusterID:f3aab545-2339-455d-b0d7-30cc88706548 StartedAt:1542247734333485131 LastUp:1542247734333485131}
I181115 02:08:56.802661 445 storage/replica_command.go:306  [n1,split,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
I181115 02:08:56.817433 12 server/testserver.go:426  had 13 ranges at startup, expected 20
I181115 02:08:56.820290 12 server/testserver.go:426  had 13 ranges at startup, expected 20
I181115 02:08:56.826725 12 server/testserver.go:426  had 13 ranges at startup, expected 20
I181115 02:08:56.827471 12 server/testserver.go:426  had 13 ranges at startup, expected 20
I181115 02:08:56.828178 12 server/testserver.go:426  had 13 ranges at startup, expected 20
I181115 02:08:56.828893 12 server/testserver.go:426  had 13 ranges at startup, expected 20
I181115 02:08:56.829811 12 server/testserver.go:426  had 13 ranges at startup, expected 20
I181115 02:08:56.830935 12 server/testserver.go:426  had 13 ranges at startup, expected 20
I181115 02:08:56.832552 12 server/testserver.go:426  had 13 ranges at startup, expected 20
I181115 02:08:56.835436 12 server/testserver.go:426  had 13 ranges at startup, expected 20
I181115 02:08:56.903793 462 storage/replica_command.go:306  [n1,split,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I181115 02:08:56.906476 12 server/testserver.go:426  had 14 ranges at startup, expected 20
I181115 02:08:56.915801 12 server/testserver.go:426  had 14 ranges at startup, expected 20
I181115 02:08:56.941770 12 server/testserver.go:426  had 14 ranges at startup, expected 20
I181115 02:08:56.994413 547 storage/replica_command.go:306  [n1,split,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
I181115 02:08:57.008277 12 server/testserver.go:426  had 15 ranges at startup, expected 20
I181115 02:08:57.081356 12 server/testserver.go:426  had 15 ranges at startup, expected 20
I181115 02:08:57.146281 563 storage/replica_command.go:306  [n1,split,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r17]
I181115 02:08:57.209246 416 storage/replica_command.go:306  [n1,split,s1,r17/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r18]
I181115 02:08:57.218309 12 server/testserver.go:426  had 17 ranges at startup, expected 20
I181115 02:08:57.308120 492 storage/replica_command.go:306  [n1,split,s1,r18/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r19]
I181115 02:08:57.432782 566 storage/replica_command.go:306  [n1,split,s1,r19/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r20]
I181115 02:08:57.490622 12 server/testserver.go:426  had 19 ranges at startup, expected 20
I181115 02:08:58.350632 557 sql/event_log.go:126  [n1,client=127.0.0.1:40740,user=root] Event: "create_database", target: 52, info: {DatabaseName:test Statement:CREATE DATABASE IF NOT EXISTS test User:root}
I181115 02:08:58.500385 558 storage/replica_command.go:306  [n1,split,s1,r20/1:/{Table/23-Max}] initiating a split of this range at key /Table/53 [r21]
I181115 02:08:58.512296 557 sql/event_log.go:126  [n1,client=127.0.0.1:40740,user=root] Event: "create_table", target: 53, info: {TableName:test.public.t Statement:CREATE TABLE test.public.t (k INT PRIMARY KEY, int1 INT, int2 INT, bool1 BOOL, bool2 BOOL, dec1 DECIMAL, dec2 DECIMAL, float1 FLOAT8, float2 FLOAT8, b BYTES) User:root}



TestDistAggregationTable/AVG/float1/5
--- FAIL: testrace/TestDistAggregationTable/AVG/float1/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/MIN/dec1/50
--- FAIL: testrace/TestDistAggregationTable/MIN/dec1/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/MIN/float1/100
--- FAIL: testrace/TestDistAggregationTable/MIN/float1/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/STDDEV/dec1/5
--- FAIL: testrace/TestDistAggregationTable/STDDEV/dec1/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/MAX/int2/50
--- FAIL: testrace/TestDistAggregationTable/MAX/int2/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/COUNT/float2/10
--- FAIL: testrace/TestDistAggregationTable/COUNT/float2/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/MAX/int1/10
--- FAIL: testrace/TestDistAggregationTable/MAX/int1/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/COUNT/int1/100
--- FAIL: testrace/TestDistAggregationTable/COUNT/int1/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/MIN/int2/100
--- FAIL: testrace/TestDistAggregationTable/MIN/int2/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/SUM/float2/50
--- FAIL: testrace/TestDistAggregationTable/SUM/float2/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/MIN/float1/50
--- FAIL: testrace/TestDistAggregationTable/MIN/float1/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/STDDEV/float1/5
--- FAIL: testrace/TestDistAggregationTable/STDDEV/float1/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/SUM/dec2/50
--- FAIL: testrace/TestDistAggregationTable/SUM/dec2/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/MAX/bool2/50
--- FAIL: testrace/TestDistAggregationTable/MAX/bool2/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/STDDEV/dec1/100
--- FAIL: testrace/TestDistAggregationTable/STDDEV/dec1/100 (0.000s)
Test ended in panic.

------- Stdout: -------
I181115 02:09:09.272675 4187 util/mon/bytes_usage.go:608  [n1] flow: bytes usage increases to 1.0 MiB (+10240)
I181115 02:09:09.344403 4212 util/mon/bytes_usage.go:608  [n1] flow: bytes usage increases to 2.0 MiB (+10240)



TestDistAggregationTable/COUNT/bool1/5
--- FAIL: testrace/TestDistAggregationTable/COUNT/bool1/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/COUNT/float1/100
--- FAIL: testrace/TestDistAggregationTable/COUNT/float1/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/SUM/int2/50
--- FAIL: testrace/TestDistAggregationTable/SUM/int2/50 (0.000s)
Test ended in panic.

------- Stdout: -------
I181115 02:09:04.460739 292 server/status/runtime.go:465  [n1] runtime stats: 315 MiB RSS, 179 goroutines, 19 MiB/6.7 MiB/38 MiB GO alloc/idle/total, 34 MiB/61 MiB CGO alloc/total, 0.0 CGO/sec, 0.0/0.0 %(u/s)time, 0.0 %gc (16x), 3.9 MiB/3.9 MiB (r/w)net



TestDistAggregationTable/MAX/float1/10
--- FAIL: testrace/TestDistAggregationTable/MAX/float1/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/SUM/float1/100
--- FAIL: testrace/TestDistAggregationTable/SUM/float1/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/BOOL_OR/bool1/50
--- FAIL: testrace/TestDistAggregationTable/BOOL_OR/bool1/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/AVG/float1/100
--- FAIL: testrace/TestDistAggregationTable/AVG/float1/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/MAX/float2/50
--- FAIL: testrace/TestDistAggregationTable/MAX/float2/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/COUNT/int1/50
--- FAIL: testrace/TestDistAggregationTable/COUNT/int1/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/MIN/dec2/50
--- FAIL: testrace/TestDistAggregationTable/MIN/dec2/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/STDDEV/float2/100
--- FAIL: testrace/TestDistAggregationTable/STDDEV/float2/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/COUNT/b/5
--- FAIL: testrace/TestDistAggregationTable/COUNT/b/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/STDDEV/int1/5
--- FAIL: testrace/TestDistAggregationTable/STDDEV/int1/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/BOOL_AND/bool2/10
--- FAIL: testrace/TestDistAggregationTable/BOOL_AND/bool2/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/AVG/float2/50
--- FAIL: testrace/TestDistAggregationTable/AVG/float2/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/AVG/dec2/100
--- FAIL: testrace/TestDistAggregationTable/AVG/dec2/100 (0.000s)
Test ended in panic.

------- Stdout: -------
I181115 02:09:00.746940 1349 util/mon/bytes_usage.go:608  [n1] flow: bytes usage increases to 1.0 MiB (+10240)



TestDistAggregationTable/STDDEV/float1/10
--- FAIL: testrace/TestDistAggregationTable/STDDEV/float1/10 (0.000s)
Test ended in panic.




TestDistAggregationTable/STDDEV/float1/50
--- FAIL: testrace/TestDistAggregationTable/STDDEV/float1/50 (0.000s)
Test ended in panic.




TestDistAggregationTable/SUM/float2/100
--- FAIL: testrace/TestDistAggregationTable/SUM/float2/100 (0.000s)
Test ended in panic.




TestDistAggregationTable/SUM/int1/5
--- FAIL: testrace/TestDistAggregationTable/SUM/int1/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/COUNT/bool2/5
--- FAIL: testrace/TestDistAggregationTable/COUNT/bool2/5 (0.000s)
Test ended in panic.




TestDistAggregationTable/MAX/float2/100
--- FAIL: testrace/TestDistAggregationTable/MAX/float2/100 (0.000s)
Test ended in panic.





```

Please assign, take a look and update the issue accordingly.

--- a/pkg/storage/split_queue.go
+++ b/pkg/storage/split_queue.go
@@ -223,7 +223,6 @@ func (sq *splitQueue) processAttempt(
 		); pErr != nil {
 			return errors.Wrapf(pErr, "unable to split %s at key %q", r, splitByLoadKey)
 		}
-		r.splitMu.splitFinder = nil
 		return nil
 	}
 	return nil

